internal/captain/command.go

```diff
@@ -499,7 +514,15 @@ func (c *Command) subCommandNames() []string {
 }
 
 func (c *Command) runner(cobraCmd *cobra.Command, args []string) error {
-	defer profile.Measure(fmt.Sprintf("captain:runner"), time.Now())
+	defer profile.Measure("captain:runner", time.Now())
+
+	if c.unstable {
+		if !c.cfg.GetBool(constants.UnstableConfig) {
+			c.out.Print(locale.Tr("unstable_command_warning", c.Name()))
+			return nil
+		}
+		c.out.Print(locale.T("beta_feature_banner"))
```

Comment:
For our internal logic please use "unstable" terminology rather than "beta". So we're consistent and it's easier to refactor if we need to.